project 2/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+   <title>Project 2</title>
+   <link rel="stylesheet" href="styles.css">
+   <link href="https://fonts.googleapis.com/css?family=Big+Shoulders+Display:500,600,800,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section>
+      <div class="content"> 
+      
+      <img src="images/海报 copy.jpg" alt="">
+      <div class="svg"> <svg id="line" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2 90.53"><defs><style>.cls-1,.cls-2{fill:none;stroke:#32302a;stroke-miterlimit:10;stroke-width:2px;}.cls-2{stroke-dasharray:11.22 11.22;}</style></defs><title>dashed</title><line class="cls-1" x1="1" x2="1" y2="6"/><line class="cls-2" x1="1" y1="17.22" x2="1" y2="78.92"/><line class="cls-1" x1="1" y1="84.53" x2="1" y2="90.53"/></svg>
+      </div>
+          <div class="text">
+            
+      <h1>MOON </h1>
+      <ul class="info">
+          <li class="info">2009</li>
+          <li class="info">97 <span>min</span></li>
+          <li class="special">DRAMA, MYSTERY, SCI-FI</li>
+          <li class="info"><svg id="star" viewBox="0 0 113.77 108.21">
+            <polygon class="star" points="56.89 0 74.47 35.62 113.78 41.33 85.33 69.06 92.05 108.21 56.89 89.72 21.73 108.21 28.44 69.06 0 41.33 39.31 35.62 56.89 0"/></polygon>
+          </svg>
+          7.9<span>/10</span></li>
+        </ul>
+        
+      <ul class="watch">
```

Comment:
Same note about the repeated classnames. You can totally use a classname on multiple items -- but it _should_ be for the same styles, which is not what you're doing here.